lib/Runtime/Library/WasmLibrary.cpp

```diff
@@ -523,7 +523,12 @@ namespace Js
             *ppSourceInfo = nullptr;
             Wasm::WasmModule * wasmModule = nullptr;
 
-            *ppSourceInfo = Utf8SourceInfo::New(scriptContext, (LPCUTF8)script, lengthBytes / sizeof(char16), lengthBytes, pSrcInfo, false);
+            *ppSourceInfo = Utf8SourceInfo::New(scriptContext, (LPCUTF8)script, lengthBytes / sizeof(char16), lengthBytes, pSrcInfo, false
+#ifndef NTBUILD
+                            , nullptr);
+#else
+                            );
+#endif
```

Comment:
Similarly, TTSanpValues.cpp and TTActionEvents.cpp can use GlobalObject.cpp style.

---
In reply to: [85626412](https://github.com/Microsoft/ChakraCore/pull/1830#discussion_r85626412) [](ancestors = 85626412)